polonius-engine/src/output/location_insensitive.rs

```diff
@@ -27,12 +27,11 @@ pub(super) fn compute<Region: Atom, Loan: Atom, Point: Atom>(
         .chain(all_facts.cfg_edge.iter().map(|&(_, q)| q))
         .collect();
 
-    all_facts
-        .region_live_at
-        .reserve(all_facts.universal_region.len() * all_points.len());
+    let mut my_region_live_at = all_facts.region_live_at.clone();
+    my_region_live_at.reserve(all_facts.universal_region.len() * all_points.len());
```

Comment:
Nit: I'd probably drop the `my_` prefix

Reply:
Fixed this in https://github.com/rust-lang/polonius/pull/102/commits/833ba373baff68404b5c52d0fcf34b6bbea1ca50, which also motivates why I think this is ok style-wise in the commit message.